contracts/ICS20Transfer.sol

```diff
@@ -41,12 +41,12 @@ contract ICS20Transfer is
     /// @dev It's implemented on a custom ERC-7201 namespace to reduce the risk of storage collisions when using with
     /// upgradeable contracts.
     /// @param escrow The escrow contract
-    /// @param ibcDenomContracts Mapping of non-native denoms to their respective IBCERC20 contracts
+    /// @param denomIDContracts Mapping of non-native denoms to their respective IBCERC20 contracts
     /// @param ics26Router The ICS26Router contract
     /// @custom:storage-location erc7201:ibc.storage.ICS20Transfer
     struct ICS20TransferStorage {
         IEscrow escrow;
-        mapping(string => IBCERC20) ibcDenomContracts;
+        mapping(bytes32 => IBCERC20) ibcDenomContracts;
```

Comment:
```suggestion
        mapping(bytes32 => IBCERC20) denomIDContracts;
```
You forgot to change the name? (Since natspec is changed) but also I don't really like the name too much.